client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -58,6 +59,24 @@ const StockListComponent: FC = () => {
     />
   );
 
+  const ItemCell = <T extends StockLineRowFragment>({
+    rowData,
+  }: CellProps<T>): React.ReactElement<CellProps<T>> => {
+    const itemName = rowData.item.name;
+    return (
+      <Tooltip title={String(itemName)} placement="bottom-start">
+        <div
+          style={{
+            overflow: 'hidden',
+            textOverflow: 'auto',
```

Comment:
Sorry! Didn't notice this before.. I really like having the ... when the text is truncated
```suggestion
            textOverflow: 'ellipsis',
```